platform/admin/core/admin-core-appservice/src/main/java/org/codice/ddf/admin/application/service/impl/ApplicationImpl.java

```diff
@@ -124,7 +124,7 @@ private void setRepoNameAndVersion(String repoName) {
             name = repoNameParts[0];
             version = repoNameParts[1];
             break;
-        case 3:
+        default:
             throw new RuntimeException(String.format("Could not tokenize repository name of '%s'",
```

Comment:
default is equivalent to case: 0 and I could rename it to that, it would probably would fail before this point because the feature name would be empty and that should never happen. I can change it to log an error since it was really annoying that this causes the app to fail to deploy.